lib/api/include/irods/apiTable.hpp

```diff
@@ -1269,6 +1271,20 @@ static irods::apidef_t client_api_table_inp[] = {
         "api_get_library_features", irods::clearInStruct_noop, irods::clearOutStruct_noop,
         (funcPtr) CALL_GET_LIBRARY_FEATURES_OUT
     },
+    {
+        GET_RESCINFO_FOR_PUT_AN, RODS_API_VERSION, REMOTE_USER_AUTH, REMOTE_USER_AUTH,
```

Comment:
I think `REMOTE_USER_AUTH` should be `LOCAL_USER_AUTH`.

Reply:
Seems fine to me, but then this must change for the `rcGetHostFor{Put|Get} `API's as well, correct?

Reply:
You're right. The current auth level you have allows remote users to call the API.